values-schema.yaml

```diff
@@ -1435,6 +1433,39 @@ definitions:
     required:
       - name
       - url
+  user:
+    type: object
+    description: A user can be a team member or an admin.
+    properties:
+      email:
+        $ref: '#/definitions/email'
+        x-secret: ''
+      firstName:
+        type: string
+        x-secret: ''
+      lastName:
+        type: string
+        x-secret: ''
+      isPlatformAdmin:
+        type: boolean
+        x-secret: ''
+      isTeamAdmin:
+        type: boolean
+        x-secret: ''
+      teams:
+        type: array
+        items:
+          type: string
+          x-secret: ''
+      initialPassword:
```

Comment:
what will happen if I create a user without initialPassword ?

Reply:
I tested this issue. If we create a user without initialPassword, the apl-keycloak-operator will create a user with an `undefined` (as string) initial password (which is not good). We should handle this in the apl-keycloak-operator and generate an initial random password before the user creation but that initial password will not stored anywhere so the user needs to reset password by platform admins. Do you have any idea? @j-zimnowoda @srodenhuis @ElderMatt

Reply:
Can bootstrap.ts do it?

Reply:
Yes, in `bootstrap.ts` we can generate a random password and add it in the users who do not have an initial password.

Reply:
`bootstrap.ts` will generate random initial passwords for users if they don't have one.

Reply:
Nice, I see passwords being generated now.